apps/react-native/.kernelrc/env/env.prod.ts

```diff
@@ -1,15 +1,30 @@
 import {Config} from '@brandingbrand/kernel-core';
 import type {KernelPluginPermissions} from '@brandingbrand/kernel-plugin-permissions';
+import type {KernelPluginFastlane} from '@brandingbrand/kernel-plugin-fastlane';
 
-const prod: Config & KernelPluginPermissions = {
+const dev: Config & KernelPluginPermissions & KernelPluginFastlane = {
   ios: {
     name: 'HelloWorld',
     bundleId: 'com.helloworld',
     displayName: 'Hello World',
+    versioning: {
+      version: '0.0.1',
+      build: 1,
+    },
+    signing: {
+      exportTeamId: "762H5V79XV",
+      exportMethod: "test",
+      provisioningProfileName: "Test Provisioning Profile",
+      profilesDir: "./xx"
+    }
   },
   android: {
     displayName: 'Hello World',
     packageName: 'com.helloworld',
+    versioning: {
+      version: '0.0.1',
+      build: 1,
```

Comment:
nothing seems to be relying on these build numbers, the fastlane file just seems to rely on fetching from app center, and this doesn't seem to get injected into android build.gradle
Should we drop it from the type? Or use it if no version yet in app center?

Reply:
yeah i should probably change the name of the android one to `versionCode`. its not necessarily relevant to app-center which is why I didn't do the version in the `kernelPluginFastlane` both version and versionCode should end up in the `app/build.gradle`

Reply:
We will need to discuss this because if there is a present build number it will override the versionCode that is fetched from app center i.e. disable that part of fastfile